### PR TITLE
Provide documentation for Azure Service Bus message pump

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide documentation for Azure Service Bus message pump.

Closes https://github.com/arcus-azure/arcus.messaging/issues/14

### CODE REVIEW

**Comment by tomkerkhove** [comment]
It's finally here @stijnmoreels!

File: docs/features/message-pumps/service-bus.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Configuration
+
+Once the message handler is created, you can very easily configure it:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    // Add Service Bus Queue message pump and use OrdersMessageHandler to process the messages
+    // ISecretProvider will be used to lookup the connection string scoped to the queue for secret ARCUS_SERVICEBUS_ORDERS_CONNECTIONSTRING
+    services.AddServiceBusQueueMessagePump<OrdersMessageHandler>("ARCUS_SERVICEBUS_ORDERS_CONNECTIONSTRING");
```
Comment (stijnmoreels) [inline docs/features/message-pumps/service-bus.md]: The syntax here will change of course; but it's OK to have it like this for now.

File: docs/features/message-pumps/service-bus.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+We support **connection strings that are scoped on the Service Bus namespace and entity** allowing you to choose the required security model for your applications. If you are using namespace-scoped connection strings you'll have to pass your queue/topic name as well.
+
+Next to that, we provide a **variety of overloads** to allow you to:
+
+- Specify the name of the queue/topic
+- Configure how the message pump should work *(ie. max concurrent calls & auto delete)*
+- Read the connection string from the configuration *(although we highly recommend using a secret store instead)*
+
+## Want to get started easy? Use our templates!
```
Comment (stijnmoreels) [inline docs/features/message-pumps/service-bus.md]: Yeah, good to mention this!

File: docs/features/message-pumps/service-bus.md (hunk 1 above)
Comment (tomkerkhove) [reply docs/features/message-pumps/service-bus.md]: Agreed, just adding this to have something for now!

File: docs/features/message-pumps/service-bus.md (hunk 2 above)
Comment (tomkerkhove) [reply docs/features/message-pumps/service-bus.md]: Why not 😁

**Pull request merged**